examples/asr/hkust/README.md

```diff
@@ -1,18 +1,21 @@
 
 # Examples for HKUST
 
-## 1 Transformer
-
 ```bash
 source env.sh
-python examples/asr/hkust/prepare_data.py /tmp-data/dataset/opensource/hkust
-python athena/main.py examples/asr/hkust/transformer.json
+bash examples/asr/hkust/run.sh
 ```
 
-## 2 MTL_Transformer_CTC
+1) Before you run `examples/asr/hkust/run.sh`, you should download the hkust dataset and store it in `/nfs/project/datasets/opensource_data/hkust`. The script `examples/asr/hkust/local/prepare_data.py` would generate the desired csv file decripting the dataset
 
-```bash
-source env.sh
-python examples/asr/hkust/prepare_data.py /tmp-data/dataset/opensource/hkust
-python athena/main.py examples/asr/hkust/mtl_transformer.json
-```
+2) With the generated csv file, we should compute the cmvn file firstly like this `python athena/cmvn_main.py examples/asr/hkust/mpc.json examples/asr/hkust/data/all.csv`
+
+3) You can perform the unsupervised pretraining using the json file `examples/asr/hkust/mpc.json` or just skip this
+
```

Comment:
Probably add one more thing: if you run Mac.json, how you should update transformer.json or mtl_transformer.json

Reply:
Sure，we may add some more results